addons/tmf_loadouts/loadouts/ussr_vdv_1980.hpp

```diff
@@ -0,0 +1,540 @@
+	vest[] = {"V_Chestrig_rgr"};
+	headgear[] = {};
+	goggles[] = {"default"};
+	primaryWeapon[] = {"srifle_LRR_F"};
+	scope[] = {"optic_SOS"};
+	sidearmWeapon[] = {"hgun_Pistol_heavy_01_F"};
+	magazines[] = {
+		LIST_7("7Rnd_408_Mag"),
+		LIST_2("HandGrenade"),
+		LIST_4("11Rnd_45ACP_Mag")
+	};
+};
+
+class sp : g {
+	displayName = "Spotter";
+	uniform[] = {"U_B_GhillieSuit"};
+	vest[] = {"V_Chestrig_rgr"};
+	headgear[] = {};
+	scope[] = {"optic_MRCO"};
+	silencer[] = {"muzzle_snds_h"};
+	items[] += {
+		"ACE_MapTools",
+		"ACE_RangeCard",
+		"ACE_Kestrel4500",
+		"ACE_ATragMX"
+	};
+	linkedItems[] = {
+		"ItemMap",
+		"ItemCompass",
+		"ItemRadio",
+		"ItemWatch",
+		"Laserdesignator",
+		"ItemGPS"
+	};
+};
+*/
+
+class vg : smg {
+	displayName = "Vehicle Gunner";
+	backpack[] = {};
+	headgear[] = {"CUP_H_RUS_TSH_4_Brown"};
+};
+
+class vc : vg {
+	displayName = "Vehicle Commander";
+	linkedItems[] += {"Binocular"};
+};
+
+class vd : vg {
+	displayName = "Vehicle Driver";
+	traits[] = {"engineer"};
+	backpack[] = {"CUP_B_CivPack_WDL"};
+	backpackItems[] = {"ToolKit"};
+};
+
+class hc : smg {
+	displayName = "Helicopter Crew";
+	uniform[] = {"CUP_U_O_RUS_Flora_1_VDV"};
+	vest[] = {"V_TacVest_khk"};
+	backpack[] = {};
+	headgear[] = {
+		"CUP_H_RUS_ZSH_Shield_Down",
+		"CUP_H_RUS_ZSH_Shield_Up"
+	};
+	items[] += {"ACE_MapTools"};
+};
+
+class hp : hc {
+	displayName = "Helicopter Pilot";
+};
+
+class hcc : hc {
+	displayName = "Helicopter Crew Chief";
+	traits[] = {"engineer"};
+	backpack[] = {"CUP_B_CivPack_WDL"};
+	backpackItems[] = {"ToolKit"};
+};
+
+class jp : smg {
+	displayName = "Jet pilot";
+	traits[] = {"engineer"};
+	primaryWeapon[] = {};
+	secondaryWeapon[] = {"CUP_hgun_PMM"};
+	uniform[] = {"CUP_U_O_SLA_Overalls_Pilot"};
+	vest[] = {"CUP_V_CDF_OfficerBelt"};
+	backpack[] = {};
+	headgear[] = {"CUP_H_RUS_ZSH_Shield_Down"};
+	goggles[] = {"CUP_G_PMC_RadioHeadset"};
+	linkedItems[] += {"ItemGPS"};
+	magazines[] = {
+		LIST_4("CUP_12Rnd_9x18_PMM_M"),
+		"SmokeShellPurple"
+	};
+};
+
+class eng : car {
+	displayName = "Combat Engineer (Explosives)";
+	traits[] = {"engineer", "explosiveSpecialist"};
+	secondaryWeapon[] = {"ACE_VMM3"};
+	backpack[] = {"CUP_B_HikingPack_Civ"};
+	items[] = {
+		"ACE_DefusalKit",
+		"ACE_EntrenchingTool",
+		"ACE_Clacker",
+		"ACE_wirecutter"
+	};
+	magazines[] += {
+		LIST_4("ClaymoreDirectionalMine_Remote_Mag")
+	};
+	backpackItems[] = {
+		"ToolKit",
+		LIST_2("DemoCharge_Remote_Mag"),
+		LIST_2("SLAMDirectionalMine_Wire_Mag")
+	};
+};
+
+class engm : car {
+	displayName = "Combat Engineer (Mines)";
+	traits[] = {"engineer", "explosiveSpecialist"};
+	secondaryWeapon[] = {"ACE_VMM3"};
+	backpack[] = {"CUP_B_HikingPack_Civ"};
+	items[] += {
+		LIST_2("APERSMine_Range_Mag"),
+		LIST_2("APERSTripMine_Wire_Mag"),
+		"ACE_DefusalKit",
+		"ACE_EntrenchingTool",
+		"ACE_Clacker",
+		"ACE_wirecutter"
+	};
+	backpackItems[] = {
+		"ToolKit",
+		"CUP_MineE_M"
+	};
+};
+
+/*
+class UAV : car {
```

Comment:
Do not comment out classes, even if they aren't used. Uncomment and simply give it the UAVHacker trait and opfor UAV terminal, remove backpack.